src/julia_threads.h

```diff
@@ -101,10 +145,11 @@ struct _jl_tls_states_t {
     struct _jl_module_t *current_module;
     struct _jl_task_t *volatile current_task;
     struct _jl_task_t *root_task;
+//#ifdef COPY_STACKS
```

Comment:
Can this be removed?

Reply:
This conditional can't be used in public facing code, but if you do delete COPY_STACKS, then this would get deleted to.